src/admin/utilities/api.ts

```diff
@@ -35,14 +33,10 @@ var refreshing = false;
 export const attachRetry = (onRequestToken: () => Promise<string | null>, onError: () => void) => {
   axiosRetry(api, {
     retryCondition: async (error: AxiosError) => {
-      if (refreshing) return false;
```

Comment:
Remove the flag that was put in for load reduction. If there are multiple requests, only one of them will be retried during refresh.